web-services/metrics/src/main/java/datawave/query/metrics/ShardTableQueryMetricHandler.java

```diff
@@ -439,8 +437,8 @@ public void updateMetric(QueryMetric updatedQueryMetric, DatawavePrincipal dataw
 
             // write new entry
             writeMetrics(updatedQueryMetric, Collections.singletonList(updatedQueryMetric), lastUpdated, false);
-        } finally {
-            enableLogs(true);
+        } catch (Exception e) {
```

Comment:
So we can't do an empty catch here b/c that means we suppress an error. To keep this functionality the same as before, we should drop the try around the code block above.

```
try {
  // some code
} finally {
}
```
Is equivalent to 
```
// some code
```